lib/engine/step/g_1889/special_track.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require_relative '../special_track'
+
+module Engine
+  module Step
+    module G1889
+      class SpecialTrack < SpecialTrack
```

Comment:
i wonder if this should just be generic in special track? does this need to be 1889 specific?

Reply:
I think 1889 is the only game we have (at least so far) where a company's special track lay can forcibly interrupt the whole flow.

I guess with the C-P in Chesapeake, once one tile is laid, it forces you to lay the second, but after buying it in, you can use it at any point. 1889's Ehime is right when sold, no other time. I couldn't think of any other companies in any other games with that when-sold timing off the top of my head.